src/parsers.js

```diff
@@ -10,6 +10,7 @@ function parseQueryStringInput (queryString) {
   var overheads = ['fee', 'tax']; // tip is a special case, it maps to tipPercent
 
   var pairs = queryString.split('&');
+  pairs = pairs.map(pair => decodeURIComponent(pair).trim());
```

Comment:
I agree, encoding is only done on the person string and pairValues[0] on people should be decoded likewise. Doing it.